chsdi/views/features.py

```diff
@@ -80,16 +85,28 @@ def view_attribute_values_geojson(request):
     return _attributes(request)
 
 
+# TODO
```

Comment:
I do not know what is to be done, but we should either remove this comment, or mention what needs to be done